client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfiguration.java

```diff
@@ -103,13 +104,15 @@
      */
     Duration backoffSlotSize();
 
-
     /** Indicates whether client-side sympathetic QoS should be enabled. */
     ClientQoS clientQoS();
 
     /** Indicates whether QosExceptions (other than RetryOther) should be propagated. */
     ServerQoS serverQoS();
 
+    /** Both per-request and global metrics are recorded in this registry. */
+    TaggedMetricRegistry taggedMetricRegistry();
```

Comment:
Feels a tiny bit weird to put this here as most the of the other fields are pure data, but this isn't the first 'behavioury' object as we've already got a `java.net.ProxySelector`, `SSLSocketFactory` and `X509TrustManager`.